library/std/src/Std/Canon.qs

```diff
@@ -583,6 +583,22 @@ operation ApplyXorInPlaceL(value : BigInt, target : Qubit[]) : Unit is Adj + Ctl
     }
     adjoint self;
 }
+
+/// # Summary
+/// Applies operation `op` to the `target` `power` times.
+/// If `power` is negative, the adjoint of `op` is used.
+/// If `power` is 0, the operation `op` is not applied.
+operation ApplyOperationPowerA<'T>(
```

Comment:
Why do we need the `A` suffix?

Reply:
It's a convention that we use in certain cases. For example, compare ApplyToEachA. We pass an operation `op` that is adjointable and this operation is also adjointable. **It probably doesn't matter now since we only have one operation**, but I anticipate that we will also add a CA version.
Assuming we have two versions A and CA, one won't be able to use ApplyOperationPowerCA version if a function 'op' is just A. But in a context that would require CA, only ApplyOperationPowerCA could be used.
This is something we discuss on occasions, but so far it's a Q# peculiarity.